src/ObjectUtils/ElfFileTest.cpp

```diff
@@ -140,11 +141,28 @@ TEST(ElfFile, HasDynsym) {
 }
 
 static ::testing::Matcher<SymbolInfo> SymbolInfoEq(std::string_view demangled_name,
-                                                   uint64_t address, uint64_t size) {
+                                                   uint64_t address, uint64_t size,
+                                                   bool is_hotpatchable) {
   return testing::AllOf(
       testing::Property("demangled_name", &SymbolInfo::demangled_name, demangled_name),
       testing::Property("address", &SymbolInfo::address, address),
-      testing::Property("size", &SymbolInfo::size, size));
+      testing::Property("size", &SymbolInfo::size, size),
+      testing::Property("is_hotpatchable", &SymbolInfo::is_hotpatchable, is_hotpatchable));
+}
+
+TEST(ElfFile, LoadPatchableFunctionEntry) {
+  const std::filesystem::path elf_path =
+      orbit_test::GetTestdataDir() / "elf_binary_with_patchable_function_entries";
+  auto elf_file_result = CreateElfFile(elf_path);
+  ASSERT_THAT(elf_file_result, HasNoError());
+  const std::unique_ptr<ElfFile>& elf_file = elf_file_result.value();
+  const auto symbols_result = elf_file->LoadDebugSymbols();
```

Comment:
It would be nice to also have a similar test for the LoadEhOrDebugFrameEntries

Reply:
Added very similar version of the test calling into LoadEhOrDebugFrameEntriesAsSymbols.

Reply:
Thanks